test/e2e/storage/testsuites/provisioning.go

```diff
@@ -120,13 +119,11 @@ func (p *provisioningTestSuite) execTest(driver TestDriver, pattern testpatterns
 		// Ginkgo's "Global Shared Behaviors" require arguments for a shared function
 		// to be a single struct and to be passed as a pointer.
 		// Please see https://onsi.github.io/ginkgo/#global-shared-behaviors for details.
-		testProvisioning(&input)
+		testProvisioning(driver, &input)
```

Comment:
Just want to make sure I understand: `BeforeEach()` runs before `It()`s, but not necessarily before `testProvisioning()`?

Reply:
PR https://github.com/kubernetes/kubernetes/pull/72434 addresses this.